lib/os.js

```diff
@@ -23,14 +23,35 @@ Object.defineProperty(exports, 'constants', {
   value: constants
 });
 
+//Returns the Architecture Node is running on
+//Example x64, x86 , mips , etc..
 exports.arch = function() {
   return process.arch;
 };
 
+//Returns platform node is running on.
+//Examples : ubuntu , windows , etc...
 exports.platform = function() {
   return process.platform;
 };
 
+//Returns root directory path for Node
+exports.nodePath = function() {
+  return process.env.NODE_PATH; 
+}
+
+//Returns root directory path for NVM
+exports.nvmPath = function() {
+  return process.env.NVM_PATH;  
+}
+
+//Returns current directory
+exports.currentDir = function() {
+ return process.cwd; 
+}
+
+//Returns temporary directory
```

Comment:
Comment makes it sound like it returns a directory that is temporary, which is not accurate. Maybe the docs describe this better?